examples/hosts/app-integration/external-data/src/mock-customer-service/service.ts

```diff
@@ -8,11 +8,32 @@ import { Server } from "http";
 import cors from "cors";
 import express from "express";
 import fetch from "node-fetch";
-import { isWebUri } from "valid-url";
 
-import { MockWebhook } from "../utilities";
 import { assertValidTaskData, TaskData } from "../model-interface";
 
+export const fluidServicePort = 7070;
```

Comment:
I would recommend defining this elsewhere in the package, since it really isn't related to the customer service (it's some constant specific to the Fluid Service as used by our app). We don't have a directory for Fluid Service concepts, so maybe just under `utilities`?